src/app/login-signup/sign-up/sign-up.component.html

```diff
@@ -53,28 +136,45 @@
 					[(ngModel)]="confirmPassword"
 					name="ConfirmInput"
 					spellcheck="false"
+					(focus)="showTooltipCP = true"
+					(blur)="showTooltipCP = false"
 				>
 
+				<app-icon
+					*ngIf="password === confirmPassword"
+					class="icon x-check check"
+					iconName="check"
+					iconSize="20"
+				></app-icon>
+
+				<app-icon
+					*ngIf="!(password === confirmPassword)"
+					class="icon x-check times"
+					iconName="times"
+					iconSize="15"
+				></app-icon>
+
 				<app-icon
 					class="icon"
 					iconName="eye"
 					iconSize="25"
 					isButton="true"
-					(click)="ConfirmInput.type = 'password' "
-					*ngIf="ConfirmInput.type == 'text' "
+					(click)="ConfirmInput.type = 'password'"
+					*ngIf="ConfirmInput.type == 'text'"
 				></app-icon>
 
 				<app-icon
 					class="icon"
 					iconName="eye-slash"
 					iconSize="25"
 					isButton="true"
-					(click)="ConfirmInput.type = 'text' "
-					*ngIf="ConfirmInput.type == 'password' "
+					(click)="ConfirmInput.type = 'text'"
+					*ngIf="ConfirmInput.type == 'password'"
 				></app-icon>
 			</div>
 		</label>
+			<!-- Confirm Password End -->
 
-		<button [disabled]="validatePassed()">Login</button>
+		<button [disabled]="!validatePassed()">Login</button>
```

Comment:
I know this is annoying, but it would be strange if this were kept in.
```suggestion
		<button [disabled]="!validatePassed()">Create Account</button>
```